.gitignore

```diff
@@ -37,3 +37,6 @@ jspm_packages
 .node_repl_history
 
 .eslintcache
+
+# Unix files
+.DS_Store
```

Comment:
This should be in your personal `.gitignore` in your home directory. We should not be striving to put OS cruft files into our `.gitignore`. Regardless, it is unrelated.

```suggestion
```